src/Npgsql/Replication/PgOutput/PgOutputReplicationOptions.cs

```diff
@@ -15,23 +15,60 @@ public class PgOutputReplicationOptions : IEquatable<PgOutputReplicationOptions>
     /// <param name="publicationName">The publication names to include into the stream</param>
     /// <param name="protocolVersion">The version of the logical streaming replication protocol</param>
     /// <param name="binary">Send values in binary representation</param>
-    /// <param name="streaming">Enable streaming of in-progress transactions</param>
+    /// <param name="streamingMode">Enable streaming of in-progress transactions</param>
     /// <param name="messages">Write logical decoding messages into the replication stream</param>
     /// <param name="twoPhase">Enable streaming of prepared transactions</param>
-    public PgOutputReplicationOptions(string publicationName, ulong protocolVersion, bool? binary = null, bool? streaming = null, bool? messages = null, bool? twoPhase = null)
-        : this(new List<string> { publicationName ?? throw new ArgumentNullException(nameof(publicationName)) }, protocolVersion, binary, streaming, messages, twoPhase)
-    { }
+    [Obsolete("Please switch to the overloads that take a PgOutputProtocolVersion value instead.")]
```

Comment:
@Brar this looks odd - these are new constructor overloads which are [Obsolete]? I think the idea was to keep the original constructor (the one with `bool? streaming` and `ulong protocolVersion`), add [Obsolete] on it and just have it call the new one...